TestRandom.java

```diff
@@ -2,5 +2,20 @@
 public  class  TestRandom {
 	public static void main(String[]  args) {
 	    // Replace this comment with your code
+		int num=0,i=0,smaller=0,bigger=0;
+		Double random=0.0,ratio=0.0;
```

Comment:
the variable **random** is used only inside the the **for loop**, so we don't need to declare it outside of the loop